src/components/DatePickerContainer/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { DateRangePicker, isInclusivelyBeforeDay } from 'react-dates' 
+import 'react-dates/lib/css/_datepicker.css'
+import './DatePickerContainer.css'
+import store from '../../store'
+import moment from 'moment'
+
+class DatePickerContainer extends React.Component {
+	static propTypes = {
+		dispatch: PropTypes.func.isRequired
+	}
+
+	constructor() {
+		super()
```

Comment:
We should pass in `props` as arguments to `constructor()` and `super()`. ([Source](https://facebook.github.io/react/docs/react-component.html#constructor))